lib/src/ui/movie_list.dart

```diff
@@ -63,66 +64,63 @@ class _MovieListState extends State<MovieList> {
           ),
         ),
       ),
-      body: Container(
-        decoration: BoxDecoration(
-          gradient: backgroundColor,
-        ),
-        child: StreamBuilder(
-          stream: widget.movieBloc.allMovies,
-          builder: (context, AsyncSnapshot<Movie> snapshot) {
-            if (snapshot.hasData) {
-              return buildList(snapshot.data!);
-            } else if (snapshot.hasError) {
-              return Text(
-                snapshot.error.toString(),
-              );
-            }
-            return Center(
-              child: CircularProgressIndicator(),
+      body: StreamBuilder(
+        stream: widget.movieBloc.allMovies,
+        builder: (context, AsyncSnapshot<Movie> snapshot) {
+          if (snapshot.hasData) {
+            return buildList(snapshot.data!);
+          } else if (snapshot.hasError) {
+            return Text(
+              snapshot.error.toString(),
             );
-          },
-        ),
+          }
+          return Center(
+            child: CircularProgressIndicator(),
+          );
+        },
       ),
     );
   }
 
   Widget buildList(Movie data) {
     return GridView.builder(
-        itemCount: data.results.length,
-        gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
-          crossAxisCount: NumericConstants.crossAxisCount,
-        ),
-        itemBuilder: (BuildContext context, int index) {
-          return InkWell(
+      itemCount: data.results.length,
+      gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+        crossAxisCount: NumericConstants.crossAxisCount,
+      ),
+      itemBuilder: (BuildContext context, int index) {
+        return OpenContainer(
+          closedColor: Colors.black,
+          transitionType: ContainerTransitionType.fadeThrough,
+          transitionDuration: Duration(seconds: 2),
```

Comment:
you should make this value a constant and add a trailing comma at the end

Reply:
Fixed